File: Models/FurtherEducation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace MtpApp.Models
{
    public class FurtherEducation
    {
        public int Id { get; set; }

        [Required]
        public int ClientId { get; set; }

        public bool University { get; set; }

        public bool PublicSchool { get; set; }

        public bool VocationalTraining { get; set; }

        public bool ComputerSchool { get; set; }

        public bool EnglishSchool { get; set; }

        public bool ChineseSchool { get; set; }
        
        [StringLength(255)]
        public string AvailableTime { get; set; }
    }
}

File: spring-backend/src/main/java/com/mtp/api/models/FurtherEducation.java
package com.mtp.api.models;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Entity
@Table(name = "FurtherEducations")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class FurtherEducation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "ClientId")
    private Client client;

    private boolean university;
    private boolean publicSchool;
    private boolean vocationalTraining;
    private boolean computerSchool;
    private boolean englishSchool;
    private boolean chineseSchool;

    @Size(max = 255)
    private String availableTime;
}
